kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConnectorConfig.java

```diff
@@ -1,82 +0,0 @@
-package com.wepay.kafka.connect.bigquery.config;
-
-/*
- * Copyright 2016 WePay, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-
-import org.apache.kafka.common.config.ConfigDef;
-import org.apache.kafka.common.config.ConfigException;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import java.util.Map;
-
-/**
- * Class for connector-specific configuration properties.
- */
-public class BigQuerySinkConnectorConfig extends BigQuerySinkConfig {
```

Comment:
What exactly is the reason for removing this?

Reply:
The only member of this config (auto create tables) has been moved to its parent (`BigQuerySinkConfig`) because I had to access this property also from the task level.
After that `BigQuerySinkConnectorConfig` became empty.